added ASO secret generation (#4)
hmcts/chart-postgresql: README.md

- To use this chart, you need a postgres Flexible server created beforehand.
- Please see [cnp-flux-config](https://github.com/hmcts/cnp-flux-config/pull/25165) PR link where the Flexible server is deployed using ASO.

> - Please follow this documentation for generating [ASO secret](https://github.com/hmcts/cnp-flux-config/blob/master/docs/secrets-sops-encryption.md)

Reviewer: ```suggestion
- Please follow this documentation for generating the [PosgreSQL secret](https://github.com/hmcts/cnp-flux-config/blob/master/docs/secrets-sops-encryption.md) and see below for an example:
```

> - Simple Example to generate ASO secret and encrypt

Reviewer: ```suggestion
```

> ```

Reviewer: ```suggestion
```bash
```

brew install sops
```

> ```

Reviewer: ```suggestion
```bash
```

> kubectl create secret generic <secret name> -n <namespace> --from-literal=<key>=<value> --type=Opaque -o yaml --dry-run=client > <name of the file>

Reviewer: something like this
```suggestion
NAMESPACE=probate # adjust this for your Kubernetes namespace
PASSWORD=$(LC_ALL=C tr -dc 'A-Za-z0-9' </dev/urandom | head -c 16 ; echo)
kubectl create secret generic postgres -n $NAMESPACE --from-literal=PASSWORD=${PASSWORD} --type=Opaque -o yaml --dry-run=client > postgres.enc.yaml
```

> kubectl create secret generic prometheus-values -n sscs --from-literal=PASSWORD=fjfuyry7e --type=Opaque -o yaml --dry-run=client > demo-postgres.enc.yaml

Reviewer: this says prometheus-values which is wrong, i've done a better command above
```suggestion
```

> sops --encrypt --azure-kv https://dcdcftappsdemokv.vault.azure.net/keys/sops-key/7a5cc0c79b02466c86bc594c431e00f7 --encrypted-regex "^(data|stringData)$" --in-place postgres.enc.yaml

Reviewer: extracted so there's no scrolling
```suggestion
KEY=https://dcdcftappsdemokv.vault.azure.net/keys/sops-key/7a5cc0c79b02466c86bc594c431e00f7
sops --encrypt --azure-kv ${KEY}  --encrypted-regex "^(data|stringData)$" --in-place postgres.enc.yaml
```

> sops --encrypt --azure-kv https://dcdcftappsdemokv.vault.azure.net/keys/sops-key/7a5cc0c79b02466c86bc594c431e00f7 --encrypted-regex "^(data|stringData)$" --in-place demo-postgres.enc.yaml

Reviewer: ```suggestion
sops --encrypt --azure-kv https://dcdcftappsdemokv.vault.azure.net/keys/sops-key/7a5cc0c79b02466c86bc594c431e00f7 --encrypted-regex "^(data|stringData)$" --in-place postgres.enc.yaml
```

- To use this chart, you need a postgres Flexible server created beforehand.
- Please see [cnp-flux-config](https://github.com/hmcts/cnp-flux-config/pull/25165) PR link where the Flexible server is deployed using ASO.
- Please follow this documentation for generating the [PosgreSQL secret](https://github.com/hmcts/cnp-flux-config/blob/master/docs/secrets-sops-encryption.md) and see below for an example:
```bash
brew install sops
```

```bash
NAMESPACE=probate # adjust this for your Kubernetes namespace
PASSWORD=$(LC_ALL=C tr -dc 'A-Za-z0-9' </dev/urandom | head -c 16 ; echo)

> kubectl create secret generic postgres -n $NAMESPACE --from-literal=PASSWORD=${PASSWORD} --type=Opaque -o yaml --dry-run=client > postgres.enc.yaml

Reviewer: ```suggestion
kubectl create secret generic postgres \
  -n $NAMESPACE --from-literal=PASSWORD=${PASSWORD} --type=Opaque -o yaml \
   --dry-run=client > postgres.enc.yaml
```

- To use this chart, you need a postgres Flexible server created beforehand.

> - Please see [cnp-flux-config](https://github.com/hmcts/cnp-flux-config/pull/25165) PR link where the Flexible server is deployed using ASO.

Reviewer: ```suggestion
- See [cnp-flux-config](https://github.com/hmcts/cnp-flux-config/pull/25165) PR link where the Flexible server is deployed using ASO.
```

> - Please follow this documentation for generating the [PosgreSQL secret](https://github.com/hmcts/cnp-flux-config/blob/master/docs/secrets-sops-encryption.md) and see below for an example:

Reviewer: ```suggestion
- Follow this documentation for generating the [PosgreSQL secret](https://github.com/hmcts/cnp-flux-config/blob/master/docs/secrets-sops-encryption.md) and see below for an example:
```

```bash
brew install sops
```

```bash
NAMESPACE=probate # adjust this for your Kubernetes namespace
PASSWORD=$(LC_ALL=C tr -dc 'A-Za-z0-9' </dev/urandom | head -c 16 ; echo)
kubectl create secret generic postgres \
  -n $NAMESPACE --from-literal=PASSWORD=${PASSWORD} --type=Opaque -o yaml \

>    --dry-run=client > postgres.enc.yaml

Reviewer: ```suggestion
  --dry-run=client > postgres.enc.yaml
```
